### PR TITLE
feat(CLAUDE.md): created for opinions and oral_args submodules

### PR DESCRIPTION
**Pull request by grossir**
#1793

Juriscraper has a lot of quirks that confuse both human users and agents.

This CLAUDE.md attempts to create guidelines for the development process (issue creation, development, code review). In a sense, it could be used by humans too.

I have tried to keep it concise and limit it to non obvious points that we value in scraper development

### CODE REVIEW

**Comment by grossir** [comment]
I think it needs some more comments on  backscraping, download_backwards, etc. It got a little confused today implementing a backscraper

File: juriscraper/opinions/CLAUDE.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+**Structure:**
+- The title should include the failing scraper ID or court name and a description. Example: "`conn` scraper failing"
+- If referencing data in the source website, include a link to the source website.
+- If referencing data in Courtlistener (date range with 0 opinions, an opinion with bad data, etc) include a Courtlistener link
+- Include error messages / stack traces from Sentry or logs
+- Reference Sentry issue IDs when applicable
+  - Sentry links let reviewers count occurrences, see affected courts, and trace exact failing records. Any proposed fix should address those specific occurrences.
+- Show data examples (screenshots, SQL counts, specific records) to illustrate the problem
+  - Reviewers need real edge cases to verify fixes against. Without examples, PRs get reviewed blindly and bugs slip through.
+  - If the issue is getting to big, use the `<details><summary>Title of collapsible section</summary>A lot of content</details>` tags to organize extra info.
```
Comment (Luis-manzur) [inline juriscraper/opinions/CLAUDE.md:16]: suggestion
```suggestion
  - If the issue is getting too big, use the `<details><summary>Title of collapsible section</summary>A lot of content</details>` tags to organize extra info.
```

File: juriscraper/opinions/CLAUDE.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+**Data quality:**
+- Verify case names are from the correct court
+- Check that docket numbers aren't actually citations or dates
+- Verify each date value is going to the correct date field (for example, date_filed vs other_dates)
+- Return empty dict `{}` from `extract_from_text` when no match
+  - Returning `{"appeal_from_str": ""}` overwrites an existing value. Returning `{}` preserves whatever CL already has.
+- Use `date_filed_is_approximate = True` when exact dates aren't available; don't set unrealistic dates
+- `"Unknown"` status when status can't be determined, not `"Published"`
+- Try to parse the most amount of data possible. If a source shows the author of an opinion, the disposition, a summary, etc; try to pick it up. If there is some interesting field you don't know where to fit in the accepted return keys, highlight it to the user so they can research.
+- If a source shows different opinion types for the same case, consider using `ClusterSite` as a base class to group those together.
```
Comment (Luis-manzur) [inline juriscraper/opinions/CLAUDE.md:90]: No mention of ClusterSite mechanics — mentions considering ClusterSite but doesn't explain how it works or when to use it vs OpinionSiteLinear.

File: juriscraper/opinions/CLAUDE.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  - A document with only a docket number is still valuable in CL. Skipping for missing non-essential fields (status, date, summary) loses the opinion entirely.
+- Prefer letting the scraper crash over silently losing data
+  - Adding error handling to an already-failed download just obscures the real problem. If the site changed, we need to know.
+
+**Example files and testing:**
+- Use real data from the source, not synthetic data
+  - Real data captures encoding quirks, edge cases, and unexpected fields that synthetic data misses.
+- Auto-generate example compare files via: `python -m unittest -v tests.local.test_ScraperExampleTest juriscraper.opinions`
+  - Manually created compare files drift from actual scraper output. The test command generates them from saved responses.
+- Test both scraper AND backscraper with `sample_caller`.
```
Comment (Luis-manzur) [inline juriscraper/opinions/CLAUDE.md:47]: consider  adding something like python sample_caller.py -c                                
  juriscraper.opinions.united_states.state.conn would help.
